dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java

```diff
@@ -39,13 +42,30 @@ public String launchInstructions(String trsID) {
     @Override
     public VersionTypeValidation validateWorkflowSet(String initialPath, String contents,
         Map<String, Pair<String, GenericFileType>> indexedFiles) {
-        VersionTypeValidation validation = new VersionTypeValidation(true, new HashMap<>());
-        //        for(String line : contents.split("\\r?\\n")) {
-        //            if (!line.startsWith("import") && !line.startsWith("author") && !line.startsWith("description")) {
-        //                validation.setValid(false);
-        //                validation.getMessage().put(initialPath, "unknown keyword");
-        //            }
-        //        }
+
+        Map<String, String> validationMessageObject = new HashMap<>();
+
+        Yaml yaml = new Yaml();
+        List<String> files;
+        boolean isValid = true;
+        try {
+            Map<String, Object> map = yaml.load(contents);
```

Comment:
You could also define a class for this; I have done it that way in branch. But this is fine.

Reply:
I figure this section is going to change a lot more as we finalise the dockstore.yml, so for now I'll keep it very basic.